### PR TITLE
Remove duplicate table entry

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR removes a duplicate table entry

#### Ticket Link
N/A

### CODE REVIEW

**Review by catalintomai — APPROVED** [review]

File: site/content/integrate/apps/call/_index.md [hunk 1]
```diff
@@ -133,7 +133,6 @@ There are several types ([godoc](https://pkg.go.dev/github.com/mattermost/matter
 | :---------------- | :----- | :------------------ |
 | `type`            | string | Use `navigate`.     |
 | `navigate_to_url` | string | URL to navigate to. |
-| `navigate_to_url` | string | URL to navigate to. |
```
Comment (hanzei) [inline site/content/integrate/apps/call/_index.md:136]: Given that you already touched the code: Can we document https://github.com/mattermost/mattermost-plugin-apps/blob/ecf4e285e1a7fb2f85b0c7a2b5870f98a22a81fb/apps/call.go#L124-L124 here?

Comment (jfrerich) [reply site/content/integrate/apps/call/_index.md:136]: done.  Please let me know if the wording is ok.  I also removed another duplicate.

File: site/content/integrate/apps/call/_index.md [hunk 2]
```diff
@@ -79,7 +79,6 @@ Depending on the location and expansions, calls will have different context. The
 | `channel_id`               | string   | ID from the channel from within which the call was performed.                       |
 | `post_id`                  | string   | ID from the post from within which the call was performed.                          |
 | `root_post_id`             | string   | If the call was performed from a post in a thread, the root post ID of that thread. |
-| `mattermost_site_url`      | string   | Mattermost base URL.                                                                |
```
Comment (hanzei) [inline site/content/integrate/apps/call/_index.md:82]: Why was this removed?

**Review by hanzei — CHANGES_REQUESTED** [review]

File: site/content/integrate/apps/call/_index.md (hunk 2 above)
Comment (jfrerich) [reply site/content/integrate/apps/call/_index.md:82]: It is duplicated in the table.

https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/content/integrate/apps/call/_index.md?plain=1#L82-L84

Comment (hanzei) [reply site/content/integrate/apps/call/_index.md:82]: Missed that

**Review by hanzei — APPROVED** [review]
